### ISSUE TITLE
Resource Report - 1.1.1 Non-Text Content - Suspicious alt text on attached images

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9759]
As a user reliant on a screen reader to interpret images, alt text associated with images should provide a clear, succinct description of the contents of the image.

**Linked Work Item:**
#9663  

**Expected Result:**
EXAMPLE: The image of a sheep on 'Test Model Sheep' resources report should have a short alt text description which informs the user that they have focused on an image of a sheep (e.g. 'Image of a sheep')

**Actual Result:**
Images attached to the Resource Report have alt text which does not give the user an accurate or informative description of the image. For example, on 'Test Model Sheep', the alt text for the attached image is 'Mabel.jpg' (just the filename); it should be a succinct description of the image.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/240c5966-d57f-46a7-b4f9-0e99bd8f5dce)

**Comment by apeters** [comment]
@imogenroberts and @phudson-he I don't think there is anything we can do about this as we don't collect any metadata about the image itself.  We may have to live with the file name for now.

**Comment by johnatawnclementawn** [comment]
Once the image datatype is added (see #9333), we can address this ticket
